CMakePlugins.cmake

```diff
@@ -10,7 +10,7 @@
 # help@hdfgroup.org.
 #
 option (PLUGIN_USE_EXTERNAL "Use External Library Building for filter PLUGIN else search" OFF)
-option (PLUGIN_USE_LOCALCONTENT "Use local file for PLUGIN FetchContent" OFF)
+cmake_dependent_option (PLUGIN_USE_LOCALCONTENT "Use local file for PLUGIN FetchContent" OFF PLUGIN_USE_EXTERNAL OFF)
```

Comment:
The naming of these two options made it a bit difficult for me to reason about this at first; the use of local content being dependent on the use of external content is kind of confusing.

Reply:
Agree, however I can't figure out better names without expanding to 100 chars